src/services/meta-coin.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { fromPromise } from 'rxjs/observable/fromPromise';
+import { Web3Service } from './web3.service'
+
+const metaincoinArtifacts = require('../../build/contracts/MetaCoin.json');
+const contract = require('truffle-contract');
+
+@Injectable()
+export class MetaCoinService {
+
+	MetaCoin = contract(metaincoinArtifacts);
+
+  constructor(
+  	private web3Ser: Web3Service,
+  	) { 
+  	// Bootstrap the MetaCoin abstraction for Use
+  	this.MetaCoin.setProvider(web3Ser.web3.currentProvider);
+  }
+
+  GetBalance(account): Observable<number> {
```

Comment:
Can this be renamed to `getBalance()`? I'd rather have the first word not capitalized for consistency. Same goes for `SendCoin`, and `GetAccounts` of `Web3Service`